test/mjs/plugins/plugin.mjs

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+export default function pluginMjs (f, opts, next) {
+  f.get('/mjs', (request, reply) => {
+    reply.send({ hello: 'world' })
+  })
+
+  next()
+}
```

Comment:
It'd be good to add a test with a module using fastify-plugin

Reply:
I‘ll do that.